.gitleaks.toml

```diff
@@ -112,7 +112,8 @@ title = "gitleaks config"
     files = ['''^\.?gitleaks.toml$''',
     '''(.*?)(jpg|gif|doc|pdf|bin)$''',
     '''(go.mod|go.sum)$''',
-    '''docusaurus.config.js$''']
+    '''docusaurus.config.js$''',
+    '''credentials.yml$''']
```

Comment:
Maybe we should prefix it with the test folder to be safe?

Reply:
@HotThoughts Joe has a good point, can you add a full path to the file?

Reply:
Updated!